forte/processors/data_augment/algorithms/word_splitting_processor.py

```diff
@@ -98,51 +98,47 @@ def default_configs(cls):
         Returns:
             A dictionary with the default config for this processor.
             Additional keys for determining how many words will be split:
-            - alpha: 0 <= alpha <= 1. indicates the percent of the words
-            in a sentence that are changed.
-        Config Values:
-
-            - `other_entry_policy` (dict):
+        The keys to this dictionary are:
+            - alpha:
```

Comment:
Several suggestions.
1. At the beginning of the list bulletpoint, there should be an empty line.
2. "-" should align with the start of the previous sentence on a vertical line, in this case "the"
3. "The keys to this dictionary are:" should have the same indentation level as the previous docstring. Or you might not even need this description as the previous docstring is clear enough. (So a simple fix for 2 and 3 is just delete "The keys to this dictionary are:")
4. You can use [building docs command](https://github.com/asyml/forte/blob/d3083beca858d7f254090c3e3185f78f2dfed110/.github/workflows/main.yml#L136) and see how you docstring look like in a html.

Reply:
Doing that gave me an unexpected indent error

Reply:
It seems like the it has passed test cases. If you still have issue you can refer to this [edit](https://github.com/asyml/forte/pull/636/commits/fd89c925c96fe879bed46bf302c9b42bb4905874#diff-b522065264cdf426e37a70fb07d873a84c5e537aa20602de61e0c50efa861fccL179).

For other related issues, you can search keyword in this [PR](https://github.com/asyml/forte/pull/636/commits) for example fix.